runtime/pipeline/publisher.yaml

```diff
@@ -0,0 +1,27 @@
+Description: AWS IAM Role for publishing layers
+
+Parameters:
+  BuilderAccount:
+    Type: String
+
+Resources:
+  LayerPublisherRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: bref-layer-publisher
+      AssumeRolePolicyDocument:
+        Statement:
+          Effect: Allow
+          Principal:
+            AWS: !Sub "arn:aws:iam::${BuilderAccount}:role/bref-php-binary-builder"
+          Action: sts:AssumeRole
+      Policies:
+        - PolicyName: LayerPublisherRole
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - lambda:PublishLayerVersion
+                  - lambda:AddLayerVersionPermission
+                Resource:
+                  - '*'
```

Comment:
Does it matter if we allow any layer to be published or just an explicit list of layers?